packages/react/src/components/ButtonGroup/README.md

```diff
@@ -0,0 +1,77 @@
+# Button Group
+
+> The button group component is to be utilized within IBM.com for grouping two
```

Comment:
I think we can remove "two" based on the latest commit.
```suggestion
> The button group component is to be utilized within IBM.com for grouping
```

Reply:
Updated language to say `two or more`